doc/manual/src/installation/upgrading.md

```diff
@@ -4,7 +4,8 @@ Multi-user Nix users on macOS can upgrade Nix by running: `sudo -i sh -c
 'nix-channel --update &&
 nix-env -iA nixpkgs.nix &&
 launchctl remove org.nixos.nix-daemon &&
-launchctl load /Library/LaunchDaemons/org.nixos.nix-daemon.plist'`
+launchctl load /Library/LaunchDaemons/org.nixos.nix-daemon.plist'`  
+If this does not work try: `nix upgrade-nix`.
```

Comment:
We should probably the old instructions if they don't work. However, I guess `nix upgrade-nix` does still require `launchtl` calls. It would be nice if it did that automatically...

Reply:
Old instructions did not worked for me. I assume it might be issue specific to my env.
Perhaps old way works for other people 🤔 

[The docs for nix upgrade-nix](https://nixos.org/manual/nix/stable/command-ref/new-cli/nix3-upgrade-nix.html) say:
```
Warning
This program is experimental and its interface is subject to change.
```

That is why I thought adding this as alternative is a good idea.

WDYT @edolstra?